frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/test-expand.scenario.ts

```diff
@@ -0,0 +1,76 @@
+import { browser, ExpectedConditions as until } from 'protractor';
+import { execSync } from 'child_process';
+import { clickKebabAction } from '@console/internal-integration-tests/views/crud.view';
+import { click } from '@console/shared/src/test-utils/utils';
+import { testName } from '@console/internal-integration-tests/protractor.conf';
+import { testPVC, testDeployment } from '../../mocks/expand-test-mocks';
+import { requestSizeValue } from '../../views/add-capacity.view';
+import {
+  goToPersistentVolumeClaims,
+  expandButton,
+  createNewPersistentVolumeClaim,
+  expandSizeOption,
+  capacityDropdown,
+} from '../../views/pvc.view';
+import { SIZE_UNITS } from '../../utils/consts';
+
+const expandValue = String(Number(testPVC.size) + 1);
+
+const createDeployment = () =>
+  execSync(`echo '${JSON.stringify(testDeployment)}' | kubectl create -f -`);
+
+describe('Tests Expand flow for PVC', () => {
+  beforeAll(async () => {
+    // Create a PVC
+    await createNewPersistentVolumeClaim(testPVC, true, createDeployment);
+    // Wait for it to bind
+    await goToPersistentVolumeClaims();
+  });
+
+  it('Test PVC can be expanded (In MiBs)', async () => {
+    await clickKebabAction(testPVC.name, 'Expand PVC');
+    await browser.wait(until.visibilityOf(requestSizeValue));
+    await requestSizeValue.sendKeys(expandValue);
+    await click(capacityDropdown);
+    await click(expandSizeOption(SIZE_UNITS.MI));
+    await click(expandButton);
+
+    const pvcJSON = JSON.parse(
+      execSync(`kubectl get pvc ${testPVC.name} -n ${testName} -o json`).toString(),
+    );
+    const requestedStorage = pvcJSON.spec.resources.requests.storage;
+    expect(requestedStorage.trim()).toEqual(`${expandValue}${SIZE_UNITS.MI}`);
```

Comment:
Shouldn't `${expandValue}${SIZE_UNITS.MI}` equals to difference of the previous and current requested capacity?

Reply:
No that's not how it works. These values are absolute. requestedStorage is always equal to the absolute value. Therefore you cannot request storage value less than the current capacity(similar to shrinking).

Reply:
No this is not required since the value is absolute. Thus resulting in the value being set to what I passed, hence you cannot pass values lower than the current storage capacity.